website/docs/d/lambda_layer_version.html.markdown

```diff
@@ -35,6 +35,7 @@ This data source supports the following arguments:
 
 This data source exports the following attributes in addition to the arguments above:
 
+* `code_sha256` - Base64-encoded representation of raw SHA-256 sum of the zip file.
```

Comment:
Should we also tweak the `source_code_hash` description so that it explicitly calls out that it's a virtual attribute available to trigger replacements when the local hash changes (versus the remote, computed hash returned by AWS)?

Reply:
@jar-b I'll update the resource docs to be more explicit and add a deprecation note on the data source